bbActRoom.js

```diff
@@ -351,7 +351,10 @@ class bbActRoom extends Phaser.Scene {
         this.room2a_E_KeyImg.y = this.room2a_character_north.y-75;
         if(this.coin0.alpha == 1.0) this.room2a_E_KeyImg.alpha = 1.0;
         if(this.room2a_key_E.isDown) {
-            if(this.coin0.alpha == 1.0) this.collectCoin(0);
+            if(this.coin0.alpha == 1.0) {
+                if(roomProgress <= 2105) roomProgress = 2105;
+                 this.collectCoin(0);
```

Comment:
collectCoin is NOT inside the if statement here...

Reply:
Okay, doesn't need to be inside if-statement of roomProgress. It's indentation is confusing though (should move left).